src/components/Navbar/Navbar.js

```diff
@@ -8,9 +8,41 @@ import { ReactComponent as AddItem } from "../../img/add_item.svg";
 import { ReactComponent as ChangeList } from "../../img/change_list.svg";
 import { ReactComponent as ShareList } from "../../img/share_list.svg";
 
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
+import swal from "@sweetalert/with-react";
 
 export default function Navbar(props) {
+  const history = useHistory();
+
+  const shareTokenHandler = () => {
+    const token = localStorage.getItem("token");
+    swal({
+      title: token,
+      text: "Use this token to share this shopping list with friends",
+      dangerMode: true,
+    });
+  };
+
+  const logoutHandler = () => {
+    swal({
+      title: "Do you want to log out?",
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willLogout) => {
+      if (willLogout) {
+        localStorage.removeItem("token");
+        props.setAuth(false);
+        history.push("/");
+        swal({
+          text: "You have successfully logged out",
+          dangerMode: true,
+          icon: "success",
```

Comment:
I like that there's a modal to show that the user successfully logged out, but I don't think we need to have the button present for them to close it.

Reply:
True! I've fixed that in this code. We could consider the same thing for the modal that pops up when deleting an item.